### ISSUE TITLE
NoMethodError: undefined method `pusher' for Grocer:Module

### ISSUE THREAD

**Issue by jspooner** [frozon/passbook#55]
I'm setting up push notifications and I'm getting a no method error for Grocer when calling Passbook::PushNotification.send_notification

``` bash
➜  wallet git:(62) ✗ rails c
Loading development environment (Rails 4.2.0)
irb: warn: can't alias context from irb_context.
2.2.0 :001 > Passbook::PushNotification.send_notification 'SNIP'
NoMethodError: undefined method `pusher' for Grocer:Module
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/passbook-0.4.1/lib/passbook/push_notification.rb:4:in `send_notification'
    from (irb):1
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/railties-4.2.0/lib/rails/commands/console.rb:110:in `start'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/railties-4.2.0/lib/rails/commands/console.rb:9:in `start'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/railties-4.2.0/lib/rails/commands/commands_tasks.rb:68:in `console'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/railties-4.2.0/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/railties-4.2.0/lib/rails/commands.rb:17:in `<top (required)>'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:274:in `require'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:274:in `block in require'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:240:in `load_dependency'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:274:in `require'
    from /Users/jspooner/v/wallet/bin/rails:8:in `<top (required)>'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:268:in `load'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:268:in `block in load'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:240:in `load_dependency'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/activesupport-4.2.0/lib/active_support/dependencies.rb:268:in `load'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/spring-1.4.0/lib/spring/commands/rails.rb:6:in `call'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/spring-1.4.0/lib/spring/command_wrapper.rb:38:in `call'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/spring-1.4.0/lib/spring/application.rb:183:in `block in serve'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/spring-1.4.0/lib/spring/application.rb:156:in `fork'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/spring-1.4.0/lib/spring/application.rb:156:in `serve'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/spring-1.4.0/lib/spring/application.rb:131:in `block in run'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/spring-1.4.0/lib/spring/application.rb:125:in `loop'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/spring-1.4.0/lib/spring/application.rb:125:in `run'
    from /Users/jspooner/.rvm/gems/ruby-2.2.0/gems/spring-1.4.0/lib/spring/application/boot.rb:18:in `<top (required)>'
    from /Users/jspooner/.rvm/rubies/ruby-2.2.0/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
    from /Users/jspooner/.rvm/rubies/ruby-2.2.0/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
    from -e:1:in `<main>'2.2.0 :002 > 
```

**Comment by brightbits** [comment]
Add the gem to your gemfile and use master version of passbox (as the latest gem version doesn't support pass phrase of push notification certificate)

**Comment by frozon** [comment]
Late but should be fixed with PR #58
